apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c

```diff
@@ -173,10 +173,12 @@ void utc_arastorage_db_exec_tc_p(void)
 		TC_ASSERT("db_exec", DB_SUCCESS(res));
 	}
 
+	memset(query, 0, QUERY_LENGTH);
```

Comment:
The two blocks are completely different . This one test removal of a relation . The relation is created again in the below block. The cleanup function removes all the relations, indexes and their files from the filesystem.
The cleanup is called in the last when all the positive and negative testcases end.